src/test/java/teammates/test/cases/action/InstructorFeedbackTemplateQuestionAddActionTest.java

```diff
@@ -0,0 +1,186 @@
+package teammates.test.cases.action;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.logic.core.FeedbackSessionsLogic;
+import teammates.test.driver.AssertHelper;
+import teammates.ui.controller.InstructorFeedbackTemplateQuestionAddAction;
+import teammates.ui.controller.RedirectResult;
+
+/**
+ * SUT: {@link teammates.ui.controller.InstructorFeedbackTemplateQuestionAddAction}.
```

Comment:
Import correctly and change this to `SUT: {@link InstructorFeedbackTemplateQuestionAddAction}.`

Reply:
what do you mean by import correctly tho? I have changed it to `SUT: {@link InstructorFeedbackQuestionAddAction}`

Reply:
That should be fine as you have imported the required directory already :+1: